compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/packaging/converters/PathConverter.java

```diff
@@ -123,7 +124,11 @@ public Path start() {
         }
     
         if (Files.isRegularFile(path)) {
-            return Stream.of(new FileSystemSourceInput(path));
+            return Stream.of(
+                    DEFAULT_PKG_PATH.equals(pkgId.name.value) ?
```

Comment:
Use `Names.DOT.equals(...)`. Don't need the `DEFAULT_PKG_PATH` constant

Reply:
Replaced with `Names.DEFAULT_PACKAGE.getValue().equals(...)`